libr/core/panels.c

```diff
@@ -881,7 +881,9 @@ R_API int r_core_visual_panels(RCore *core) {
 			if (curnode == 1) {
 				r_core_cmd0 (core, "s-8");
 			} else {
-				panels[curnode].sy--;
+				if(panels[curnode].sy > 0) {
```

Comment:
please use the space between `if` and `(`

Reply:
i fixed this now.